watertap/examples/flowsheets/electrodialysis/tests/test_electrodialysis_1stack.py

```diff
@@ -0,0 +1,262 @@
+        }
+        for arc, port_tpl in arc_dict.items():
+            assert arc.source is port_tpl[0]
+            assert arc.destination is port_tpl[1]
+
+        # Test the primary EDstack properties
+        # test configrations
+        assert len(m.fs.EDstack.config) == 13
+        assert not m.fs.EDstack.config.dynamic
+        assert not m.fs.EDstack.config.has_holdup
+        assert m.fs.EDstack.config.operation_mode == "Constant_Voltage"
+        assert (
+            m.fs.EDstack.config.material_balance_type == MaterialBalanceType.useDefault
+        )
+        assert (
+            m.fs.EDstack.config.momentum_balance_type
+            == MomentumBalanceType.pressureTotal
+        )
+        assert m.fs.EDstack.config.property_package is m.fs.properties
+        assert "H2O" in m.fs.properties.component_list
+
+        # test all essential params and vars are built
+        assert isinstance(m.fs.EDstack.membrane_set, Set)
+        assert isinstance(m.fs.EDstack.cell_pair_num, Var)
+        assert isinstance(m.fs.EDstack.cell_width, Var)
+        assert isinstance(m.fs.EDstack.cell_length, Var)
+        assert isinstance(m.fs.EDstack.spacer_thickness, Var)
+        assert isinstance(m.fs.EDstack.membrane_thickness, Var)
+        assert isinstance(m.fs.EDstack.solute_diffusivity_membrane, Var)
+        assert isinstance(m.fs.EDstack.ion_trans_number_membrane, Var)
+        assert isinstance(m.fs.EDstack.water_trans_number_membrane, Var)
+        assert isinstance(m.fs.EDstack.water_permeability_membrane, Var)
+        assert isinstance(m.fs.EDstack.membrane_areal_resistance, Var)
+        assert isinstance(m.fs.EDstack.current_applied, Var)
+        assert isinstance(m.fs.EDstack.current_density_x, Var)
+        assert isinstance(m.fs.EDstack.voltage_applied, Var)
+        assert isinstance(m.fs.EDstack.voltage_x, Var)
+        assert isinstance(m.fs.EDstack.current_utilization, Var)
+        assert isinstance(m.fs.EDstack.diluate.power_electrical_x, Var)
+        assert isinstance(m.fs.EDstack.specific_power_electrical, Var)
+        assert isinstance(m.fs.EDstack.current_efficiency_x, Var)
+
+        assert isinstance(m.fs.EDstack.eq_get_total_areal_resistance_x, Constraint)
+        assert isinstance(m.fs.EDstack.eq_get_current_density, Constraint)
+        assert isinstance(m.fs.EDstack.eq_get_voltage_x, Constraint)
+        assert isinstance(m.fs.EDstack.eq_mass_transfer_term_diluate, Constraint)
+        assert isinstance(m.fs.EDstack.eq_mass_transfer_term_concentrate, Constraint)
+        assert isinstance(m.fs.EDstack.eq_power_electrical, Constraint)
+        assert isinstance(m.fs.EDstack.eq_specific_power_electrical, Constraint)
+        assert isinstance(m.fs.EDstack.eq_current_efficiency_x, Constraint)
+        assert isinstance(m.fs.EDstack.eq_isothermal_diluate, Constraint)
+        assert isinstance(m.fs.EDstack.eq_isothermal_concentrate, Constraint)
```

Comment:
You don't need to test all of these things in the flowsheet test, if these were to fail the unit model tests would catch that.

You should try and only test the things that the flowsheet adds/does